site/src/Librecores/ProjectRepoBundle/Resources/views/Default/delete.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}Delete project - LibreCores{% endblock %}
+
+{% block content %}
+<h1>The following project has been deleted</h1>
+
+<h2><a href="{{ path('librecores_project_repo_project_view', {'parentName':
```

Comment:
Two comments on that one:

First, `h2` is used for a headline, and the text is no headline. In addition, you can make the text a bit more descriptive. How about something like this? (Add links and content as appropriate.)

```html
<h1>Project deleted</h1>
Your project xxx has been deleted from LibreCores.

Do you want to add a new project? Or search for an existing one?
```

Second, there there's no point in linking to a project that has just been deleted.

Reply:
Yeah i agree, I have modified as per this agreed layout.